app/models/post.rb

```diff
@@ -1,4 +1,11 @@
 class Post < ApplicationRecord
   belongs_to :user
+  validates :title, presence: true, length: {maximum:255}
+
+  validates :body, presence: true, length: {minimum:10}
   has_many :comments
+
+  validates :user_id, presence: true
```

Comment:
I believe this is not necessary because the `belong_to` association requires you to have a `user_id` to be able to create a Post record.